.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,32 @@
-## What changes were proposed in this pull request?
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
+2. Ensure you have added or run the appropriate tests for your PR: https://spark.apache.org/developer-tools.html
+3. Understand the relationship beteween JIRAs to file issues and PRs to propose some changes for them, as documented in guidelines.
+4. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP][SPARK-XXXX] Your PR title ...'
+5. Update your PR description whenever it is needed.
+6. Provide a minimised example to reproduce the issue if possible.
+-->
 
-(Please fill in changes proposed in this fix)
+### What changes were proposed in this pull request?
 
-## How was this patch tested?
+### Why changes are needed?
 
-(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
-(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)
+### Special notes for your reviewer
+<!--
+This special notes can be any notes for reviewers to review faster and more efficiently. For instance, see the examples below.
+1. If you refactor some codes with changing classes, showing the class hierarchy will help reviewers.
+2. If you fix some SQL features, you can provide some references of other DBMSes.
+3. If there is a design documentation, please leave the link.
```

Comment:
Ideally 3 and 4 should be either JIRA link or another section of PR's content even this section is not introduced.